### PR TITLE
More changes

### PR DESCRIPTION
**Pull request by karlzipser**
Changes in most of the files. Some things are renamed. all_steer becomes valid_data_moments, for example.

### CODE REVIEW

File: Train.py [hunk 1]
```diff
@@ -22,12 +22,29 @@
 criterion = torch.nn.MSELoss().cuda()
 optimizer = torch.optim.Adadelta(net.parameters())
 
+"""
```
Comment (raunakdoesdev) [inline Train.py:25]: We are currently making the transition to saving and resuming the train loss saves using the standard csv format. I already opened a pull request with this but @tpankaj is making some changes before it will be merged in. For now, I think the current untested resuming method without saving implemented should be removed from the codebase.

File: Train.py [hunk 2]
```diff
@@ -72,14 +89,19 @@
...
                 save_timer.reset()
             if print_timer.check():
-                print('mode=' + mode)
-                print('ctr=' + str(data_index.ctr))
-                print('epoch progress=' + str(100 * data_index.ctr /
-                                              len(data_index.all_steer)) + '%')
+                print(d2n('mode=',mode,
+                    ',ctr=',data_index.ctr,
+                    ',epoch progress=',dp(100*data_index.ctr /(len(data_index.valid_data_moments)*1.0)),
+                    ',epoch=',data_index.epoch_counter))
```
Comment (raunakdoesdev) [inline Train.py:95]: This fixes #5

File: save/test_run/README.md [hunk 3]
```diff
@@ -9,3 +9,7 @@ Dataset used was:
 ```
 bair_car_data_Main_Dataset
 ```
+
+
+# 7/12/17 KZ: When I use above command, data are saved into training/save. Please indicate command for saving to some version of training/save/test_run
```
Comment (raunakdoesdev) [inline save/test_run/README.md:14]: I updated this README file with the proper command in a recent commit. Sorry about the confusion.

**Comment by raunakdoesdev** [comment]
Apparently there are some merge conflicts, someone may have made a change to Train.py that conflicts with one of your changes. Do you mind running the following, merging and commiting to your fork?

```
git fetch upstream
git merge upstream/master
# Fix merge conflict in Train.py
git commit
git push
```
